set.mm

```diff
@@ -170889,23 +170892,23 @@ seq m ( x. , G ) ~~> z ) ) $.
       VUIVUGVUEVUIVUGRVUFCFYMBYAYBYCYNYMYSYDYEYIYMBJCYJYGYKYL $.
   $}
 
-  $( Cancellation of law for exponential function.  Equation 27 of [Rudin]
-     p. 164.  (Contributed by NM, 13-Jan-2006.) $)
+  $( Cancellation law for exponential function.  Equation 27 of [Rudin] p. 164.
+     (Contributed by NM, 13-Jan-2006.) $)
   efcan $p |- ( A e. CC -> ( ( exp ` A ) x. ( exp ` -u A ) ) = 1 ) $=
     ( cc wcel cneg caddc co ce cfv cmul wceq negcl efadd mpdan cc0 negid fveq2d
     c1 ef0 syl6eq eqtr3d ) ABCZAADZEFZGHZAGHUBGHIFZQUAUBBCUDUEJAKAUBLMUAUDNGHQU
     AUCNGAOPRST $.
 
-  $( The exponential function never vanishes.  Corollary 15-4.3 of [Gleason]
-     p. 309.  (Contributed by NM, 13-Jan-2006.)  (Revised by Mario Carneiro,
-     29-Apr-2014.) $)
+  $( The exponential of a complex number is nonzero.  Corollary 15-4.3 of
+     [Gleason] p. 309.  (Contributed by NM, 13-Jan-2006.)  (Revised by Mario
+     Carneiro, 29-Apr-2014.) $)
```

Comment:
I would like to have the explanation/introduction of the term/noun "exponential" in the comment of df-ef. Maybe: The value `( e `` x )` of the exponential function at a complex number ` x ` is called "exponential (at ` x ` )".

Reply:
Ok, done for that one, but I won't add it for "sine of A" and others.